dev-python/axolotl-curve25519/axolotl-curve25519-0.4.1_p2.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver
+
+MY_PN="python-${PN}"
```

Comment:
Hmm, actually, why aren't you using the upstream name? We have a number of packages starting with `dev-python/python-`, and this package is also named `python-axolotl` on pypi, so I think that'd be a more correct name here.

Reply:
Okay, changed to python-*.